content/systems/filecoin_files/piece/_index.md

```diff
@@ -3,24 +3,62 @@ title: Piece
 weight: 2
 bookCollapseSection: true
 dashboardWeight: 1.5
-dashboardState: wip
-dashboardAudit: n/a
+dashboardState: stable
+dashboardAudit: 0
 dashboardTests: 0
 ---
 
-# Piece - Part of a file
+# The Filecoin Piece
+---
+
+The _Filecoin Piece_ is the main _unit of negotiation_ for data that users store on the Filecoin network. The Filecoin Piece is _not a unit of storage_, it is not of a specific size, but is upper-bounded by the size of the _Sector_. A Filecoin Piece can be of any size, but if a Piece is larger than the size of a Sector that the miner supports it has to be split into more Pieces so that each Piece fits into a Sector.
 
 A `Piece` is an object that represents a whole or part of a `File`,
-and is used by `Clients` and `Miners` in `Deals`. `Clients` hire `Miners`
-to store `Pieces`. 
+and is used by `Storage Clients` and `Storage Miners` in `Deals`. `Storage Clients` hire `Storage  Miners` to store `Pieces`. 
 
-The piece data structure is designed for proving storage of arbitrary
+The Piece data structure is designed for proving storage of arbitrary
 IPLD graphs and client data. This diagram shows the detailed composition
-of a piece and its proving tree, including both full and bandwidth-optimized
-piece data structures.
-
+of a Piece and its proving tree, including both full and bandwidth-optimized
+Piece data structures.
 
 ![Pieces, Proving Trees, and Piece Data Structures](pieces.png)
 
+## Data Representation
+
+It is important to highlight that data submitted to the Filecoin network go through several transformations before they come to the format at which the `StorageProvider` stores it.
+
+Below is the process followed from the point a user starts preparing a file to store in Filecoin to the point that the provider produces all the identifiers of Pieces stored in a Sector.
+
+The first three steps take place on the client side.
+
+1. When a client wants to store a file in the Filecoin network, they start by producing the IPLD DAG of the file. The hash that represents root node of the DAG is an IPFS-style CID, called _Payload CID_.
```

Comment:
It might be worth synchronizing items 1 and 2 with this conversation https://github.com/filecoin-project/specs/pull/1151#discussion_r488746195 about how the whole IPLD+CAR frame isn't strictly a requirement, it's just how Lotus works currently and how the retrieval semantics work, but I don't believe any of this is strictly built into the piece requirements. Maybe that just means softening the language so it suggest the common path, not the required path.